src/app/02-form/input/input.component.html

```diff
@@ -75,13 +76,17 @@ <h1>Input</h1>
   </kal-form-field>
 
   <kal-form-field>
-    <kal-checkbox [(ngModel)]="clearable" label="clearable"></kal-checkbox>
+    <kal-checkbox [(ngModel)]="clearable" label="clearable"></kal-checkbox>{{clearable}}
```

Comment:
is this intended ? clearable interpolation ?

Reply:
No it is a remnant of me testing things and forgeting to remove it